bin/migrate-oats-data/applications/decisions/app_modifications/application_modifications_init.py

```diff
@@ -0,0 +1,148 @@
+from common import (
+    OATS_ETL_USER,
+    setup_and_get_logger,
+    add_timezone_and_keep_date_part,
+    AlcsNoiModificationOutcomeCodeEnum,
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor, execute_batch
+
+etl_name = "init_application_modifications"
+logger = setup_and_get_logger(etl_name)
+
+
+@inject_conn_pool
+def init_application_modifications(conn=None, batch_size=1000):
+    """
+    This function is responsible for initializing the application_modification in ALCS.
+
+    Args:
+    conn (psycopg2.extensions.connection): PostgreSQL database connection. Provided by the decorator.
+    batch_size (int): The number of items to process at once. Defaults to BATCH_UPLOAD_SIZE.
+    """
+
+    logger.info(f"Start {etl_name}")
+    with conn.cursor(cursor_factory=RealDictCursor) as cursor:
+        with open(
+            "applications/decisions/sql/modifications/application_modification_insert_count.sql",
+            "r",
+            encoding="utf-8",
+        ) as sql_file:
+            count_query = sql_file.read()
+            cursor.execute(count_query)
+            count_total = dict(cursor.fetchone())["count"]
+        logger.info(f"Total Application Modifications data to insert: {count_total}")
+
+        failed_inserts = 0
+        successful_inserts_count = 0
```

Comment:
One has _count and one doesn't but both are counts, I'm fine with either but it should be consistent

Reply:
done